example/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.4'
 
 services:
   dmss:
-    image: datamodelingtool.azurecr.io/dmss:v1.8.0
+    image: datamodelingtool.azurecr.io/dmss:v1.10.0
```

Comment:
might as well set these to "latest"

Reply:
We had a discussion on this earlier this summer (when you were away) and decided then not to use latest anymore. But we could revisit this.